test/core/Vault.js

```diff
@@ -387,7 +393,9 @@ describe("Vault", function () {
       await settingsManager.setMaxOpenInterestPerSide(false, SHORTMaxOpenInterest);
       await settingsManager.setMaxOpenInterestPerUser(USERMaxOpenInterest);
       await settingsManager.setEnableDeposit(usdc.address, true);
+      await settingsManager.setEnableWithdraw(usdc.address, true);
       await settingsManager.setEnableStaking(usdc.address, true);
+      await settingsManager.setEnableUnstaking(usdc.address, true);
```

Comment:
we need to add a test for `settingsManager.setEnableUnstaking(usdc.address, false);` and make sure the unstaking revert